backend/src/openarchiefbeheer/destruction/api/viewsets.py

```diff
@@ -162,6 +179,12 @@ def create(self, request, *args, **kwargs):
     def update(self, request, *args, **kwargs):
         return super().update(request, *args, **kwargs)
 
+    def perform_destroy(self, instance: DestructionList) -> None:
+        instance.processing_status = InternalStatus.queued
```

Comment:
Do we want some sanity checking here to validate if the list is in the correct status and has all the checkmarks etc?